libeos-updater-util/tests/checkpoint.c

```diff
@@ -89,6 +89,69 @@ test_force_no_follow (Fixture                  *fixture,
 
   g_assert_false (euu_should_follow_checkpoint (fixture->sysroot, "os/eos/amd64/latest2", "os/eos/amd64/latest3", &reason));
   g_assert_cmpstr (reason, !=, NULL);
+
+  g_unsetenv ("EOS_UPDATER_FORCE_FOLLOW_CHECKPOINT");
```

Comment:
Probably wouldn't be a bad idea to run this also in `setup()` to ensure it's not inherited from the environment for tests that don't set it.